noodles-editor/public/noodles/world-flights/noodles.json

```diff
@@ -29,7 +29,7 @@
       "type": "FileOp",
       "data": {
         "inputs": {
-          "url": "@/data.json"
+          "url": "https://raw.githubusercontent.com/keplergl/kepler.gl-data/master/world_flights/world_flights_soei4h.json"
```

Comment:
Prior to agents learning from our examples I would say it's generally best to separate data used in examples from the project repo anyways because iterating on large data files like this bloats git it overtime. Now instead, I wonder how fewer examples using this pattern will influence their generation 😅. 

I think it'll be fine since we can have a few small data loading examples and have the pattern documented too. This is probably the kind of thing a rubric / prompt judge running on changes could give signal on